examples/binary/func.js

```diff
@@ -9,4 +9,4 @@ fdk.handle(async function (buf, ctx) {
   }
   ctx.responseContentType = 'application/octet-stream'
   return fdk.rawResponse(buf)
-}, {inputMode: 'buffer'})
+}, { inputMode: 'buffer' })
```

Comment:
I am new to this project. Are these from your linter/prettifier?

Reply:
I have not found a config file for one yet. Will take a deeper look later. Maybe should add one to the project if we dont have an official one already. Not necessarily in this commit though.

Reply:
the project uses the nodeJS Standard formatting guide (https://www.npmjs.com/package/standard) which lints and applies coding standards ,

 as part of the update this was upgrade to the latest version which adds several new validations.

Reply:
Cheers.